Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM eclipse-temurin:17-jdk-alpine AS builder
-WORKDIR application
+WORKDIR /application
```

Comment:
was this intentional?

Reply:
Yes we docker gave a message that not relative path should be avoided.